packages/documentation-framework/templates/html.ejs

```diff
@@ -3,7 +3,6 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width,initial-scale=1,shrink-to-fit=no">
-    <meta name="color-scheme" content="light dark">
```

Comment:
This metadata indicates that dark mode as well as light mode is supported, why remove it?

Reply:
This was creating an issue where the system preference was dictating the color of checkboxes and radio buttons, superseding the actual selected theme.

Reply:
Talked to @mcoker about it and reverting this is fine, we'll have to get back to a proper fix at a later time :+1: